index.php

```diff
@@ -0,0 +1,54 @@
+<head>
+<link rel="stylesheet" type="text/css" href="normalize.css">
+<link rel="stylesheet" type="text/css" href="style.css">
+<link href="https://fonts.googleapis.com/css?family=Fjalla+One&display=swap" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css?family=Questrial&display=swap" rel="stylesheet">
+<title>My Collection</title>
+</head>
+<body>
+    <h1>My Handbag Collection</h1>
+    <nav>
+        <div class="navbar">
+            <a class="underline">Collection</a>
+            <a class="underline">Add Item</a>
+            <a class="underline">Edit Item</a>
+        </div>
+    </nav>
+<main>
+</main>
+</body>
+<?php
+$db = new PDO('mysql:host=db; dbname=MyCollection','root','password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
```

Comment:
You might want to move the two lines above to a function - it was recommended by Ash during my code review and it's likely that you'll have to connect to a database again at some point during the next story